.github/workflows/maintenance.yml

```diff
@@ -0,0 +1,139 @@
+name: Maintenance
```

Comment:
Call it `Examples/tests maintenance (npm install, prettier, eslint, etc)`

Reply:
done